banking/src/main/java/rocks/kata/OperationType.java

```diff
@@ -0,0 +1,7 @@
+package rocks.kata;
+
+public enum OperationType {
```

Comment:
żeby to nie było takie anemiczne, proponuję zamianę `AccountOperation` na enuma

https://www.baeldung.com/a-guide-to-java-enums

Reply:
Do omówienia jutro jak do tego podejść, bo nie udało mi się zamienić `AccountOperation` na enum class. Wymagałoby to zdefiniowania konstruktora zależnego tylko od znanych stałych, a ja tu przekazuję parametry do konstruktora. Jedyne co to przerzuciłam tego enuma do klasy `AccountOperation`

Reply:
Implementacja interfejsu z metodą o odpowiedniej sygnaturze i potraktowanie wartości enuma jako strategii (singletonów).